2023-08-23T10:25:23.797Z error: Error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials m6-20020a1709061ec600b00992e14af9b9sm9591091ejj.134 - gsmtp
    at SMTPConnection._formatError (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:1564:34)
    at SMTPConnection.<anonymous> (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:969:20)
    at SMTPConnection._onData (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10)
2023-08-23T10:25:23.811Z error: Error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials m6-20020a1709061ec600b00992e14af9b9sm9591091ejj.134 - gsmtp
    at SMTPConnection._formatError (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:1564:34)
    at SMTPConnection.<anonymous> (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:969:20)
    at SMTPConnection._onData (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10)
2023-08-23T10:25:23.819Z error: Error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials q24-20020aa7cc18000000b0052567e6586bsm9209378edt.38 - gsmtp
    at SMTPConnection._formatError (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:1564:34)
    at SMTPConnection.<anonymous> (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:969:20)
    at SMTPConnection._onData (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10)
2023-08-23T10:25:23.825Z error: Error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials e11-20020a056402104b00b0051e22660835sm9199032edu.46 - gsmtp
    at SMTPConnection._formatError (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:1564:34)
    at SMTPConnection.<anonymous> (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:969:20)
    at SMTPConnection._onData (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10)
2023-08-23T10:25:23.831Z error: Error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials k14-20020a170906158e00b0099cb1a2cab0sm9653722ejd.28 - gsmtp
    at SMTPConnection._formatError (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:790:19)
    at SMTPConnection._actionAUTHComplete (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:1564:34)
    at SMTPConnection.<anonymous> (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:546:26)
    at SMTPConnection._processResponse (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:969:20)
    at SMTPConnection._onData (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:755:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\6man\projects\Lead-mailer\node_modules\nodemailer\lib\smtp-connection\index.js:193:44)
    at TLSSocket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at TLSSocket.Readable.push (node:internal/streams/readable:228:10)
2023-08-23T10:29:31.607Z error: TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.all (<anonymous>)
    at file:///C:/Users/6man/projects/Lead-mailer/index.mjs:31:19
    at Layer.handle [as handle_request] (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\6man\projects\Lead-mailer\node_modules\body-parser\lib\read.js:137:5
2023-08-24T13:32:23.463Z error: undefined
2023-08-24T13:33:34.654Z error: undefined
2023-08-28T15:23:54.055Z error: ReferenceError: JSONdata is not defined
    at file:///C:/Users/6man/projects/Lead-mailer/index.js:37:16
    at Layer.handle [as handle_request] (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\6man\projects\Lead-mailer\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
2023-08-28T15:24:28.705Z error: ReferenceError: JSONdata is not defined
    at file:///C:/Users/6man/projects/Lead-mailer/index.js:37:16
    at Layer.handle [as handle_request] (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\6man\projects\Lead-mailer\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\6man\projects\Lead-mailer\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
2023-08-28T15:57:37.215Z error: TypeError: Cannot read properties of undefined (reading 'id')
    at file:///C:/Users/6man/projects/Lead-mailer/index.js:64:13
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
2023-09-05T19:45:08.277Z error: Error: connect EHOSTUNREACH 2606:4700::6812:bce4:443
    at AxiosError.from (file:///home/apps/Lead-mailer/node_modules/axios/lib/core/AxiosError.js:89:14)
    at RedirectableRequest.handleRequestError (file:///home/apps/Lead-mailer/node_modules/axios/lib/adapters/http.js:593:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (/home/apps/Lead-mailer/node_modules/follow-redirects/index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at TLSSocket.socketErrorListener (node:_http_client:502:9)
    at TLSSocket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-09-05T20:59:31.572Z error: Error: connect EHOSTUNREACH 2606:4700::6812:bce4:443
    at AxiosError.from (file:///home/apps/Lead-mailer/node_modules/axios/lib/core/AxiosError.js:89:14)
    at RedirectableRequest.handleRequestError (file:///home/apps/Lead-mailer/node_modules/axios/lib/adapters/http.js:593:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (/home/apps/Lead-mailer/node_modules/follow-redirects/index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at TLSSocket.socketErrorListener (node:_http_client:502:9)
    at TLSSocket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-09-08T14:29:39.094Z error: Error: connect EHOSTUNREACH 2606:4700::6812:bce4:443
    at AxiosError.from (file:///home/apps/Lead-mailer/node_modules/axios/lib/core/AxiosError.js:89:14)
    at RedirectableRequest.handleRequestError (file:///home/apps/Lead-mailer/node_modules/axios/lib/adapters/http.js:593:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (/home/apps/Lead-mailer/node_modules/follow-redirects/index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at TLSSocket.socketErrorListener (node:_http_client:502:9)
    at TLSSocket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-09-08T17:46:19.956Z error: Error: connect EHOSTUNREACH 2606:4700::6812:bde4:443
    at AxiosError.from (file:///home/apps/Lead-mailer/node_modules/axios/lib/core/AxiosError.js:89:14)
    at RedirectableRequest.handleRequestError (file:///home/apps/Lead-mailer/node_modules/axios/lib/adapters/http.js:593:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (/home/apps/Lead-mailer/node_modules/follow-redirects/index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at TLSSocket.socketErrorListener (node:_http_client:502:9)
    at TLSSocket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-09-08T18:49:59.252Z error: Error: connect EHOSTUNREACH 2606:4700::6812:bce4:443
    at AxiosError.from (file:///home/apps/Lead-mailer/node_modules/axios/lib/core/AxiosError.js:89:14)
    at RedirectableRequest.handleRequestError (file:///home/apps/Lead-mailer/node_modules/axios/lib/adapters/http.js:593:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (/home/apps/Lead-mailer/node_modules/follow-redirects/index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at TLSSocket.socketErrorListener (node:_http_client:502:9)
    at TLSSocket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
2023-09-19T07:01:51.492Z error: Error: connect ETIMEDOUT 2606:4700::6812:bde4:443
    at AxiosError.from (file:///home/apps/Lead-mailer/node_modules/axios/lib/core/AxiosError.js:89:14)
    at RedirectableRequest.handleRequestError (file:///home/apps/Lead-mailer/node_modules/axios/lib/adapters/http.js:593:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (/home/apps/Lead-mailer/node_modules/follow-redirects/index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at TLSSocket.socketErrorListener (node:_http_client:502:9)
    at TLSSocket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
undefined error: TypeError: Cannot read properties of undefined (reading 'Vorname')
    at createJSONData (file:///C:/Users/6man/projects/1.WORKSET/Lead-mainler/src/JSONData.mjs:3:28)
    at file:///C:/Users/6man/projects/1.WORKSET/Lead-mainler/index.js:21:22
    at Layer.handle [as handle_request] (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\body-parser\lib\read.js:137:5
undefined error: TypeError: Cannot read properties of undefined (reading 'userData')
    at createJSONData (file:///C:/Users/6man/projects/1.WORKSET/Lead-mainler/src/JSONData.mjs:3:19)
    at file:///C:/Users/6man/projects/1.WORKSET/Lead-mainler/index.js:21:22
    at Layer.handle [as handle_request] (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\body-parser\lib\read.js:137:5
undefined error: TypeError: Cannot read properties of undefined (reading 'userData')
    at createJSONData (file:///C:/Users/6man/projects/1.WORKSET/Lead-mainler/src/JSONData.mjs:3:19)
    at file:///C:/Users/6man/projects/1.WORKSET/Lead-mainler/index.js:21:22
    at Layer.handle [as handle_request] (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\body-parser\lib\read.js:137:5
undefined error: TypeError: Cannot read properties of undefined (reading 'userData')
    at createJSONData (file:///C:/Users/6man/projects/1.WORKSET/Lead-mainler/src/JSONData.mjs:3:19)
    at file:///C:/Users/6man/projects/1.WORKSET/Lead-mainler/index.js:21:22
    at Layer.handle [as handle_request] (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\express\lib\router\index.js:280:10)
    at C:\Users\6man\projects\1.WORKSET\Lead-mainler\node_modules\body-parser\lib\read.js:137:5
undefined error: Unauthorized access
undefined error: [object Object] пізда пайп драйв
undefined error:  пізда пайп драйв
undefined error: {"success":false,"error":"unauthorized access","errorCode":401,"error_info":"Please check developers.pipedrive.com"}
undefined error: {"success":false,"error":"unauthorized access","errorCode":401,"error_info":"Please check developers.pipedrive.com"}
undefined error: {"success":false,"error":"unauthorized access","errorCode":401,"error_info":"Please check developers.pipedrive.com"}
undefined error: {"success":false,"error":"unauthorized access","errorCode":401,"error_info":"Please check developers.pipedrive.com"}
undefined error: {"success":false,"error":"unauthorized access","errorCode":401,"error_info":"Please check developers.pipedrive.com"}
undefined error: AxiosError: Request failed with status code 401
    at settle (file:///C:/Users/6man/projects/1.WORKSET/Lead-mainler/node_modules/axios/lib/core/settle.js:19:12)
    at IncomingMessage.handleStreamEnd (file:///C:/Users/6man/projects/1.WORKSET/Lead-mainler/node_modules/axios/lib/adapters/http.js:572:11)
    at IncomingMessage.emit (node:events:529:35)
    at endReadableNT (node:internal/streams/readable:1368:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
